crates/primitives/src/explorers/blockscout.rs

```diff
@@ -30,6 +30,15 @@ impl BlockScout {
             },
         })
     }
+
+    pub fn new_hyper_evm() -> Box<Self> {
+        Box::new(Self {
+            meta: Metadata {
+                name: "Hyperliquid EVM Explorer",
```

Comment:
```suggestion
                name: "BlockScout",
```

let's reuse the name